Patch by Robert Scheck <robert@fedoraproject.org> for Zarafa >= 7.0.0, which avoids the
warning in syslog regarding a not started zarafa-licensed if "./configure --enable-oss"
was used to build Zarafa or if the "licensed" directory in the source directory did not
exist at build time. This patch should not affect any of the official non-OSS builds by
Zarafa itself.

--- zarafa-7.0.0/configure.ac				2011-01-20 21:30:52.000000000 +0100
+++ zarafa-7.0.0/configure.ac.oss_only			2011-01-23 02:02:18.238695312 +0100
@@ -147,6 +147,9 @@
    AC_MSG_RESULT([ok])
 fi
 AM_CONDITIONAL([OSS_ONLY], [test "$want_oss" = yes])
+if test "$want_oss" = "yes"; then
+  AC_DEFINE(OSS_ONLY, [1], [Define to 1 if the opensource mode is enabled])
+fi
 
 AC_ARG_ENABLE(debug, AC_HELP_STRING([--enable-debug],[enable debug mode (Adds debug code)]),
   [want_debug=${enableval}],[want_debug=no])
--- zarafa-7.0.0/provider/server/ECServer.cpp		2011-01-21 00:07:36.000000000 +0100
+++ zarafa-7.0.0/provider/server/ECServer.cpp.oss_only	2011-01-23 02:10:24.954734490 +0100
@@ -907,7 +907,9 @@
 	lpLicense = new ECLicenseClient(g_lpConfig->GetSetting("license_socket"), atoui(g_lpConfig->GetSetting("license_timeout")) );
 
 	if(lpLicense->GetSerial(strSerial, lstCALs) != erSuccess) {
+#ifndef OSS_ONLY    // No warning if it's opensource mode
 	    g_lpLogger->Log(EC_LOGLEVEL_FATAL, "WARNING: zarafa-licensed not running, commercial features will not be available until it's started.");
+#endif
 	} else {
 		if (!strSerial.empty())
 			g_lpLogger->Log(EC_LOGLEVEL_FATAL, "Using commercial license serial '%s'", strSerial.c_str());
